experimental/packages/opentelemetry-instrumentation/package.json

```diff
@@ -47,7 +47,9 @@
     "tdd": "npm run tdd:node",
     "tdd:node": "npm run test -- --watch-extensions ts --watch",
     "tdd:browser": "karma start",
-    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts' --exclude 'test/browser/**/*.ts'",
+    "test:cjs": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts' --exclude 'test/browser/**/*.ts'",
+    "test:esm": "node -e 'process.exit(process.versions.node.split(\".\")[0] >= 12)' || nyc node --loader=import-in-the-middle/hook.mjs node_modules/.bin/_mocha test/node/*.test.mjs",
+    "test": "npm run test:cjs && npm run test:esm",
```

Comment:
I think most users run `npm test` when developing locally. Do we really want the tests run twice every time? Might be better to test cjs only by default and have a `test:ci` script to run both in our CI?

Reply:
I also think the `test:esm` script wont work in windows so this breaks the basic `npm test` script for windows users.

Reply:
> I also think the test:esm script wont work in windows so this breaks the basic npm test script for windows users.

`test:esm` should work for everyone has its just running with the test with the IITM hook. 

> I think most users run npm test when developing locally. Do we really want the tests run twice every time?

I think it's important for user make modification to the instrumentation logic to make sure it works with esm and cjs. The test take less than a second to run so i think its fine but happy to change this if needed.